rclpy/rclpy/wait_for_message.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from rclpy.node import Node
+from rclpy.utilities import timeout_sec_to_nsec
+from rclpy.impl.implementation_singleton import rclpy_implementation as _rclpy
+from rclpy.signals import SignalHandlerGuardCondition
+
+
+def wait_for_message(
+    msg_type,
+    node: 'Node',
+    topic: str,
+    time_to_wait=-1
+):
+    """
+    Wait for the next incoming message.
+
+    :param msg_type: message type
+    :param node: node you've created
+    :param topic: topic name you've defined
+    :time_to_wait: the timeout before returning
```

Comment:
It would be nice to add units like second or mili-second.

```suggestion
    :time_to_wait: seconds to wait before returning
```